GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -204,7 +205,14 @@ private void OnGuiLoadingScreenSelect()
 
             if (GUILayout.Button("Create Loading Scene", GUILayout.Width(ButtonWidth)))
             {
-                PlayInstantLoadingScreenGenerator.GenerateLoadingScreenScene();
+                if (string.IsNullOrEmpty(QuickDeployConfig.Config.assetBundleUrl))
```

Comment:
It's not ideal to separate the precondition check for assetBundleUrl from the place assetBundleUrl is accessed. Consider passing assetBundleUrl into GenerateLoadingScreenScene as a parameter.

Reply:
Resolved.